src/namespace/adminnamespace.js

```diff
@@ -26,16 +26,97 @@ export class AdminNamespace extends CustomNamespace {
         this.nsp.on('connection', (socket) => {
             logger.debug(`New connected socket (socketid: ${socket.id}, username: ${socket.username})`);
 
-            socket.on('newDebate', (callback) => {
+            socket.on('getDebates', (callback) => {
+                logger.debug(`Get debate requested from ${socket.username}`);
+
+                if (!(callback instanceof Function)) {
+                    logger.debug(`callback is not a function.`);
+                    return;
+                }
+
+                // TODO: Only return debates available for this user
+
+                let debates = [];
+                for (const [key, debate] of this.activeDebates.entries()) {
+                    debates.push({
+                        debateId: debate.debateID,
+                        title: debate.title,
+                        description: debate.description
+                    })
```

Comment:
End of instruction char ';' is missing.
```suggestion
                    });
```